web/src/SidebarResources.tsx

```diff
@@ -190,10 +199,65 @@ export function SidebarListSection(props: SidebarSectionProps): JSX.Element {
   )
 }
 
+const defaultMaxItems = 20
+
+const ShowMoreRow = styled.li`
+  margin: ${SizeUnit(0.5)} ${SizeUnit(0.5)} 0 ${SizeUnit(0.5)};
+  color: ${Color.grayLight};
+  font-size: ${FontSize.small};
+  display: flex;
+  align-items: center;
+  justify-content: right;
+  font-family: ${Font.sansSerif};
+`
+
+const ShowMoreButton = styled(InstrumentedButton)`
+  ${mixinResetButtonStyle};
+  font-size: ${FontSize.small};
+  color: ${Color.offWhite};
+  transition: color ${AnimDuration.default} ease;
+  cursor: pointer;
+  padding: 0 0.5em;
+
+  &:hover {
+    color: ${Color.blue};
+  }
+`
+
 function SidebarListSectionItems(props: SidebarSectionProps) {
+  let [maxItems, setMaxItems] = useState(defaultMaxItems)
+  let items = props.items
```

Comment:
(tiny nit) could you rename this var so it doesn't echo the prop name, something like `itemsToDisplay`? it took me a second to realize this was a var that's potentially getting reassigned elsewhere, rather than the prop data.

Reply:
done